src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -335,6 +378,43 @@ var ChangeEventPlugin = {
 
   eventTypes: eventTypes,
 
+  _isInputEventSupported: isInputEventSupported,
+
+  willDeleteListener(inst) {
```

Comment:
I am not sure these hooks are appropriate for this sort of thing, but they seem like they are well suited for it, baring the below annoyance